text/3-entrees/off-chain-workers/transactions.md

```diff
@@ -44,26 +42,33 @@ phase, as shown by `Entering off-chain workers` in the node output message:
 ...
 ```
 
-Referring to the code at
+If we refer to the code at
 [`pallets/offchain-demo/src/lib.rs`](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/offchain-demo/src/lib.rs),
 there is an `offchain_worker` function inside `decl_module!`. This is the entry point of the
-off-chain worker that is executed once after each block generation, so we put all the off-chain
-logic here.
+off-chain worker which is executed once per block generation, so all the off-chain
+logic is here.
 
 Two kinds of transactions can be sent back on-chain from off-chain workers, **Signed Transactions**
 and **Unsigned Transactions**. Signed transactions are used if the transaction requires the sender
-to be specified. Unsigned transactions are used when the sender does not need to be known, and
-additional logic is written in the code to provide extra data verification. Let's walk through how
-to set up each one.
+to be specified. Unsigned transactions are used when the sender does not need to be known and
+additional logic in the code provides additional data verification. 
```

Comment:
```suggestion
to be specified. Unsigned transactions are used when the sender does not need to be known, but this would require additional logic be in place to provide extra data verification. 
```